api/resources_portal/config/common.py

```diff
@@ -43,13 +43,19 @@ class Common(Configuration):
         "django.contrib.auth.middleware.AuthenticationMiddleware",
         "django.contrib.messages.middleware.MessageMiddleware",
         "django.middleware.clickjacking.XFrameOptionsMiddleware",
+        "resources_portal.middleware.oauth.OAuthMiddleWare",
     )
 
     ALLOWED_HOSTS = ["*"]
     ROOT_URLCONF = "resources_portal.urls"
     SECRET_KEY = os.getenv("DJANGO_SECRET_KEY")
     WSGI_APPLICATION = "resources_portal.wsgi.application"
 
+    # OAuth
+    CLIENT_SECRET = os.getenv("OAUTH_CLIENT_SECRET")
+    CLIENT_ID = "APP-2AHZAK2XCFGHRJFM"
+    OAUTH_URL = os.getenv("OAUTH_ENIVRONMENT", "https://sandbox.orcid.org/oauth/token")
```

Comment:
This is defined in local and production, so this should get overwritten anyway. It's also confusing that it's loading in `OAUTH_ENVIRONMENT` for `OAUTH_URL`.

Reply:
My bad. So I can just remove this line entirely?

Reply:
I think so!

Reply:
Sweet, it works. Good catch!